server/game/StatusToken.js

```diff
@@ -1,12 +1,46 @@
-const GameObject = require('./GameObject');
+const EffectSource = require('./EffectSource');
+const AbilityDsl = require('./abilitydsl');
 
-class StatusToken extends GameObject {
+class StatusToken extends EffectSource {
     constructor(game, card, isHonored) {
         super(game, isHonored ? 'Honored Token' : 'Dishonored Token');
         this.honored = !!isHonored;
         this.dishonored = !isHonored;
         this.card = card;
         this.type = 'token';
+        this.persistentEffects = [];
+
+        this.applyHonorEffects();
+    }
+
+    applyHonorEffects() {
+        if(!this.card) {
+            return;
+        }
+        let amount = this.honored ? card => card.getGlory() : card => 0 - card.getGlory();
+        let effect = {
+            match: this.card,
+            effect: AbilityDsl.effects.modifyBothSkills(amount)
```

Comment:
This needs to be a dynamic effect, otherwise changes in glory won't be reflected

Reply:
modifyBothSkills is a `flexible` effect and I am passing a function so it should be dynamic - unless I am missing what you are saying.

Reply:
yes, I just can't read.... :)